en/docs/get-started/about-this-release.md

```diff
@@ -12,227 +12,58 @@ For more information on WSO2 API Manager, see the [overview]({{base_path}}/getti
 
-??? note "Publishing third-party APIs on the Developer Portal"
+??? note "Registry artifact view support for MI Management API/Dashboard"
+    A new page has been added to the MI-Dashboard to view the `<MI-HOME>/registry` directory and registry metadata, properties and content.
 
-    WSO2 API Manager provides the capability to publish APIs that are deployed in an external Gateway. Since these APIs are not deployed in the WSO2 API Gateway, the API will not be proxied through the Gateway.
+    REST API resources have been added to the Micro Integrator as a **management API** to facilitate the functionalities of fetching existing registry data, adding new registry artifacts, modifying and deleting existing registries.
 
-    **[Learn more]({{base_path}}/deploy-and-publish/publish-on-dev-portal/third-party-api-support/)**
+??? note "Gateway Support for Single Usage Token"
+    The JWT access tokens supplied by the gateway can be used to access the dedicated API resource as many times as the user requires. However, if the application developer wishes to limit this functionality by allowing just one chance to access the resource, One Time Token can be utilized. A One Time Token (JWT) is a JWT that will be revoked after a single use. This functionality is only accessible for JWT access tokens.
 
-??? note "A read-only mode for API Publisher"
+??? note "Import/Export of Operation Policies with APICTL"
+    As an upgrade from the current version of APICTL (4.1.0), a command to export the shared operation policies in particular has been added.
 
-    This feature allows a user who has view/read-only permissions to only view the API, API Product, and Service details in the Publisher portal. This user can also review all the analytics-related details.
+??? note "Handling accidental deletions of shared backend certificates"
+    Backend certificates are currently shared amongst APIs based on endpoint. As a result, another person who creates an API with the same endpoint can view and remove a certificate that does not belong to the user. As a result, this new functionality is designed to limit the ability to delete backend certificates.
 
-    **[Learn more]({{base_path}}/design/api-security/authorization/publisher-portal-in-read-only-mode/)**
+??? note "API Linters for validating the API Specifications"
+    This feature integrates an API Linter with the support of custom rules.
 
-??? note "Mock API implementation support considering API definition in Choreo Connect"
+??? note "Generate API Documentation Automatically from OAS"
+    This feature generates API Documentation automatically using the API’s Open API Specification.
 
-    This Mock API Implementation allows you to generate mock responses based on the examples provided in the OpenAPI specification that will mock the API responses.
+??? note "Throttle policy export import support for API-M"
```

Comment:
Let's rename the title to "Support for managing Rate Limiting Policies with APICTL"